### DISCUSSION TITLE
[V3] - Search Focus

### DISCUSSION THREAD

**Discussion by xreal** [filamentphp/filament#7360] [General]
I just clicked around a bit in the demo (https://demo.filamentphp.com/)

If you enter something in the search (table filter), the input box loses focus after a short delay. I think it would be nice if the focus stayed on the search field until the user clicks away.

And i noticed that the spinner keeps spinning when nothing is found.

https://github.com/filamentphp/filament/assets/1552214/b1f0e878-6bb6-4bac-a361-b26a80b3e8f0

**Comment by pboivin** [comment]
It's been reported on Discord but I'm not sure there is a Github issue yet. You can create one if you have some time 👍️

**Comment by danharrin** [comment]
This was fixed already this morning

**Comment by cpbret** [comment]
Has the fix gone out? Is this related?

I'm on 3.0.5

https://github.com/filamentphp/filament/assets/64978571/e8bbce33-f518-471a-bec8-ba4f03a16686

**Reply by xreal** [reply]
I think the input focus problem ist fixed, as dan has written. The loading spinner problem is still there, i will take a look tomorrow and create a issue.

**Reply by xreal** [reply]
I had the same problem. I fixed it now. 
Problem in my case was that i had a custom setup with "@livewireScriptConfig". 
I load my Custom Alpine Plugins now in the base app.js like so.


```
import customPlugin from 'plugin/plugin';
document.addEventListener('alpine:init', () => {
  window.Alpine.plugin(plugin);
});

```

Hope this helps.
